.gitignore

```diff
@@ -1,4 +1,7 @@
 node_modules
 dist
 publish
+.DS_Store
+dest
```

Comment:
What's `dest`?

Reply:
the gulpfile.js had it as destination folder from the begining I just add the .DS_Store so you can remove the ./dest/ if you wish . 
![gulpfile](https://user-images.githubusercontent.com/12359920/34576921-2bde8690-f145-11e7-856c-71770cf3e223.png)